lieer/local.py

```diff
@@ -259,9 +228,10 @@ def initialize_repository (self, replace_slash_with_dot, account):
       raise Local.RepositoryException ("'mail' exists: this repository seems to already be set up!")
 
     self.state = Local.State (self.state_f)
-    self.state.replace_slash_with_dot = replace_slash_with_dot
-    self.state.account = account
-    self.state.write ()
+    self.state.write()
+    self.config.set(self.gmailieer.args.account, "replace_slash_with_dot", "true" if replace_slash_with_dot else "false")
+    with open(self.config_f, mode="w") as fd:
+      self.config.write(fd)
```

Comment:
How would this work with a non-writable config file? If I'm reading this right, any time the user specifies these values in the root config, they're copied to the account section and immediately written. This wouldn't exactly work if the config file can't be written.

Reply:
You're right, this needs to be fixed - I'll do it soon

Reply:
I've put in a try-catch block to address this.